File: programming-concepts/input-output/python/range_check.py
# Raspberry Pi Foundation
# Developed to be used alongside Isaac Computer Science,
# part of the National Centre for Computing Education
# Usage licensed under CC BY-SA 4

def main():
    user_input = input("Enter a week of the year: ")
    week = int(user_input)

    if week > 0 and week <= 52:
        print(f"You have chosen week {week}")
    else:
        print("A week must be between 1-52")

    
# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == '__main__':
    main()

File: programming-concepts/input-output/c-sharp/range_check.cs
/*
Isaac Computer Science
Usage licensed under the Open Government Licence v3.0

Note: This file is designed to be copied out and compiled on your machine.
In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
To run this file you need to:
1. Copy the contents
2. Paste them into the C# IDE of your choice (Visual Studio, for example)
3. Change the namespace to match your project (if necessary)
4. Compile the program
5. Run the program
*/

using System;

namespace IsaacCodeSamples
{

    class Variables
    {
        // The Main method is the entry point for all C# programs
        public static void Main() {
            Console.WriteLine("Enter a week of the year: ");
            string userInput = Console.ReadLine();
            int week = Int32.Parse(userInput);

            if (week > 0 && week <= 52) {
                Console.WriteLine($"You have chosen week {week}");
            }
            else {
                Console.WriteLine("A week must be between 1-52");
            }
        }
        

    }
}